mk/mcu/STM32F4.mk

```diff
@@ -107,27 +112,31 @@ INCLUDE_DIRS    := $(INCLUDE_DIRS) \
 
 ifeq ($(PERIPH_DRIVER), HAL)
 CMSIS_SRC       :=
-INCLUDE_DIRS    := $(INCLUDE_DIRS) \
-                   $(STDPERIPH_DIR)/Inc \
-                   $(USBCORE_DIR)/Inc \
-                   $(USBCDC_DIR)/Inc \
-                   $(CMSIS_DIR)/Include \
-                   $(CMSIS_DIR)/Device/ST/STM32F4xx/Include \
-                   $(SRC_DIR)/drivers/mcu/stm32/vcp_hal
+INCLUDE_DIRS    := \
```

Comment:
Rest of file uses 12 spaces (3 tabs) indentation.

Reply:
Sorted.